locations/spiders/toyota_tw.py

```diff
@@ -0,0 +1,21 @@
+from locations.categories import Categories, apply_category
+from locations.json_blob_spider import JSONBlobSpider
+from locations.spiders.toyota_us import ToyotaUSSpider
+
+
+class ToyotaTWSpider(JSONBlobSpider):
+    name = "toyota_tw"
+    item_attributes = ToyotaUSSpider.item_attributes
+    start_urls = ["https://www.toyota.com.tw/api/location.ashx"]
+    locations_key = "DATA"
+
+    def post_process_item(self, item, response, location: dict):
+        item["website"] = "https://www.toyota.com.tw/"
```

Comment:
I wouldn't include this same website on all locations, if they have no individual website then just leave it out. Instead, make sure it is on Wikidata, with relevant "applies to jurisdiction".

Reply:
Will do.

Reply:
... Toyota wikidata page is semi-protected and I do not yet have enough kudos points :-(